ccd-definition/RoleToAccessProfiles/RoleToAccessProfiles-CUI-nonprod.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "RoleName" : "Claimant",
+    "Authorisation": "",
+    "CaseTypeID": "CIVIL",
+    "AccessProfiles": "CLAIMANT",
+    "ReadOnly": "N",
+    "Disabled": "F",
+    "CaseAccessCategories": ""
```

Comment:
should not these have access to spec claim

Reply:
Please change name for Access Profile
to something like
CITIZEN_DEFENDANT_PROFILE
CITIZEN_CLAIMANT_PROFILE

And yes CaseAccessCategories need to be SPEC_CLAIM for us

Reply:
Also role name is
"RoleName" : "[DEFENDANT]",
and 
"RoleName" : "[CLAIMANT]",

Reply:
done